test/event/staking/staking-factory-events.ts

```diff
@@ -10,13 +10,18 @@ type RewardToken = {
 
 export type ActualStakingPoolCreatedEvent = {
     stakingPool: string
-    treasury: string
     creator: string
+    config: Config
+}
+type Config = {
+    treasury: string
     rewardTokens: RewardToken[]
     stakeToken: string
-    epochStartTimestamp: BigNumber
-    epochDuration: BigNumber
+    epochStartTimestamp: number
+    epochDuration: number
     minimumContribution: BigNumber
+    minTotalPoolStake: BigNumber
```

Comment:
@octavioamu max & min pool stakes are needed, correct?

Reply:
yes I basically use for the UI all the data on the struct 
```
library StakingPoolLib {
    enum RewardType {
        NONE,
        FIXED,
        FLOATING
    }

    struct Reward {
        IERC20 tokens;
        uint256 maxAmount;
        uint256 ratio; // only initialized for fixed
    }

    struct Config {
        uint256 daoId;
        uint128 minTotalPoolStake;
        uint128 maxTotalPoolStake;
        uint128 minimumContribution;
        uint32 epochDuration;
        uint32 epochStartTimestamp;
        address treasury;
        IERC20 stakeToken;
        Reward[] rewardTokens;
        RewardType rewardType;
    }
}
```